### PR TITLE
Logger - Added missing MessageTemplateFormatMethodAttribute + removed obsolete internal method

### PR DESCRIPTION
**Pull request by snakefoot**
Follow up to #2345 and #2406

### CODE REVIEW

File: src/NLog/Logger-generated.tt [hunk 1]
```diff
@@ -189,7 +189,7 @@ namespace NLog
         {
             if (Is<#=level#>Enabled)
             {
-                WriteToTargets(LogLevel.<#=level#>, message, exception);
+                WriteToTargets(LogLevel.<#=level#>, exception, message, null);
```
Comment (304NotModified) [inline src/NLog/Logger-generated.tt]: why is this change needed?

Comment (snakefoot) [reply src/NLog/Logger-generated.tt]: Because it referenced an obsolete internal method, that I have also removed (Leftovers from the reordering of the Exception-parameter to the front).

**Comment by 304NotModified** [comment]
ok thx!
